packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -287,9 +287,9 @@ String getMissingPackageHintForPlatform(TargetPlatform platform) {
 }
 
 class DebugConnectionInfo {
-  DebugConnectionInfo({ this.port, this.wsUri, this.baseUri });
+  DebugConnectionInfo({ this.httpUri, this.wsUri, this.baseUri });
```

Comment:
Agreed. Added a TODO and will be cleaned up as part of https://github.com/flutter/flutter/issues/7050